src/map/storage.c

```diff
@@ -276,13 +344,17 @@ int storage_delitem(struct map_session_data* sd, int n, int amount)
  *   0 : fail
  *   1 : success
  *------------------------------------------*/
-int storage_add_from_inventory(struct map_session_data* sd, int index, int amount)
+int storage_add_from_inventory(struct map_session_data* sd, struct storage_data *stor, int index, int amount)
 {
+	const struct storage_settings *stst = storage->get_settings(stor->uid);
```

Comment:
missing nullpo check for stor variable.
Check after this line is useless, because here server already crashed